README.md

```diff
@@ -16,9 +16,7 @@ Install into a virtual environment (Python 3.6.1+).
 
     python3 -m venv venv
     . venv/bin/activate
-    pip3 install git+ssh://git@github.com/relaypro/relay-py.git#egg=relay-py
-    cd relay-py
-    pip3 install -e .
+    pip3 install git+https://git@github.com/relaypro/relay-py.git#egg=relay-py
```

Comment:
I suggest using pip from the venv.  Upgrade it first.  So my steps are:

$ python3 -m venv venv
$ . venv/bin/activate
(venv)$ pip install -U pip
(venv)$ pip install git+https://git@github.com/relaypro/relay-py.git#egg=relay-py

Reply:
Good idea. Will do.